.github/workflows/ci.yml

```diff
@@ -306,9 +306,6 @@ jobs:
               echo "$sum $file" > ${file#\*}.sha256
             fi
           done
-      -
-        name: License
-        run: cp packaging/* ./bin/release/
```

Comment:
Should this one be swapped for the top-level `LICENSE`, or was a license already added as part of one of the steps?

Reply:
AFAIK compose is the only docker artifact where we include a LICENSE file with built artifact (for legacy reasons)
Licensing is already set by github project and source code archive

Reply:
Ah, right; I wasn't sure if folks would expect it to be there.

I know / recall it's included in the `.deb` and/or `.rpm` packages but that's probably separate from this?